### PR TITLE
Showing ProgressDialog at the time of bluetooth pairing

### LINKED ISSUE #969: Showing ProgressBar or ProgressDialog at the time of pairing other device with bluetooth
### Current Situation:

https://user-images.githubusercontent.com/87614560/206856789-de0d9d29-33a9-4f7b-ba80-eb60f543c561.mp4

So, we see that when pairing has started nothing will happen for some time. It may create confusion for the user. 

### How can we improve it?

Instead of this, if we show a ProgressDialog like some other cases in the app or show a progress bar, I think it may be more user-friendly.

If possible, then I want to work on this issue with the guidance of the maintainers.

### PR DESCRIPTION
**Pull request by SanjaySargam**
Fixes #969 

**ScreenRecording**

https://user-images.githubusercontent.com/65113071/223193999-09b75057-a828-48da-8b8c-1906e0a6f65e.mp4

### CODE REVIEW

**Comment by SebaDro** [comment]
Hi @SanjaySargam . Thanks for your contribution. 

I really like the solution for showing a progress bar when pairing performs. Just a small suggestion: we already have a [DialogUtils ](https://github.com/enviroCar/enviroCar-app/blob/master/org.envirocar.app/src/org/envirocar/app/views/utils/DialogUtils.java) helper class that provides some methods to create a `MaterialAlertDialogBuilder` with progress bar. It would be great, if you could use this helper class to create the dialog, since it also applies the enviroCar theme on the dialog, which preserves consistence in the UX

What do you think about it?

**Review by SebaDro — CHANGES_REQUESTED** [review]
Hi @SanjaySargam . Thanks for your contribution. 

I really like the solution for showing a progress bar when pairing performs. Just a small suggestion: we already have a [DialogUtils ](https://github.com/enviroCar/enviroCar-app/blob/master/org.envirocar.app/src/org/envirocar/app/views/utils/DialogUtils.java) helper class that provides some methods to create a `MaterialAlertDialogBuilder` with progress bar. It would be great, if you could use this helper class to create the dialog, since it also applies the enviroCar theme on the dialog, which preserves consistence in the UX

What do you think about it?

**Comment by SanjaySargam** [comment]
> Hi @SanjaySargam . Thanks for your contribution.
> 
> I really like the solution for showing a progress bar when pairing performs. Just a small suggestion: we already have a [DialogUtils ](https://github.com/enviroCar/enviroCar-app/blob/master/org.envirocar.app/src/org/envirocar/app/views/utils/DialogUtils.java) helper class that provides some methods to create a `MaterialAlertDialogBuilder` with progress bar. It would be great, if you could use this helper class to create the dialog, since it also applies the enviroCar theme on the dialog, which preserves consistence in the UX
> 
> What do you think about it?

Ok I will use existing Dialog in helper class

File: org.envirocar.app/res/drawable/baseline_bluetooth_connected_24.xml
Comment (SebaDro) [inline org.envirocar.app/res/drawable/baseline_bluetooth_connected_24.xml:1]: I'd rather expect the Bluetooth icon to be white, as every ProgressDialog has a white icon.

**Review by SebaDro — CHANGES_REQUESTED** [review]
Please, change the color of the Bluetooth icon to be white and select the develop branch as target branch for your pull request. After that I will merge it.

**Comment by SanjaySargam** [comment]
> Please, change the color of the Bluetooth icon to be white and select the develop branch as target branch for your pull request. After that I will merge it.

@SebaDro Ok 
To achieve this, I need to delete this PR and make another one right?

**Comment by SebaDro** [comment]
Well, I think it's possible to edit the present pull request and change the target branch. If not, just create a new one and close the old.

**Review by SebaDro — APPROVED** [review]

**Pull request merged**